Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```
@@ -40,10 +40,20 @@
               amount: 2
               doAfter: 2
 
+        - to: diagonalWall # L5 Buildable diagonal walls
```

I think there's a missing line here: 
```suggestion
        - to: diagonalWall # L5 Buildable diagonal walls
          completed:
```